Source/SwiftLintFramework/Rules/Performance/ReduceBooleanRule.swift

```diff
@@ -0,0 +1,51 @@
+import SourceKittenFramework
+
+public struct ReduceBooleanRule: Rule, ConfigurationProviderRule, AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "reduce_boolean",
+        name: "Prefer allSatisfy or contains over reduce(true) or reduce(false)",
```

Comment:
This is a bit verbose compared to our usual rule names. Can we change this to “Reduce Boolean”?

Reply:
Sure thing